packages/lib-react-components/src/Media/Media.stories.js

```diff
@@ -28,6 +28,8 @@ const VIDEO_URL =
   'https://static.zooniverse.org/fem-assets/home-video.mp4'
 const TEXT_URL =
   'https://panoptes-uploads.zooniverse.org/subject_location/f5506d1c-a0e9-4aba-a418-6a6c46a7731a.txt'
+const VOLUMETRIC_URL = 
+  'https://panoptes-uploads-staging.zooniverse.org/subject_location/3c56e6a7-be27-4ab3-93a7-c41a3c867baf.json'
```

Comment:
Double checking - is this subject structured with `{ type: 'volumetric'}`?

Is the subject url in `subjectMock.js`, which is used for all of the stories, structured as expected with `{ type: 'volumetric' }`? I noticed that file is not edited in this PR, so it could be another source of `useVolumetricSubject()` failing to find the url from `subject.locations` if the mocked subject object is structured differently than an actual subject passed to the viewer(s).